cupyx/profiler/_time.py

```diff
@@ -26,6 +26,14 @@ def __init__(self, name, ts, devices):
         self._ts = ts
         self._devices = devices
 
+    def __repr__(self) -> str:
+        """ Returns a string representation of the object.
+
+        Returns:
+            str: A string representation of the object.
+        """
+        return f'name: {self.name}, ts: {self._ts}, devices: {self._devices}'
```

Comment:
Unfortunately this does not give any information of the GPU time/ CPU time etc. Ideally we would like `__repr__` to just call `to_string`

Reply:
The call to `to_str(self, show_gpu=False)` method must be with `show_gpu=False` or `True`?